pkg/kv/dynamodb/stats.go

```diff
@@ -10,4 +10,8 @@ var (
 		Name: "dynamo_consumed_capacity_total",
 		Help: "The capacity units consumed by operation.",
 	}, []string{"operation"})
+	dynamoSlowdown = promauto.NewCounterVec(prometheus.CounterOpts{
+		Name: "dynamo_slowdown_total",
+		Help: "The number of times this operation was slowed down due to throttling.",
+	}, []string{"operation"})
```

Comment:
Do we have the total count?  Otherwise this might not be very useful...

Reply:
Total count of what? KV requests? we do :)